### PR TITLE
Controller update

### PR DESCRIPTION
**Pull request by samsep**
ping @appirio-tech/demandside

### CODE REVIEW

File: example/scripts/mock.coffee [hunk 1]
```diff
@@ -2,18 +2,10 @@ AutoConfigFakeServer.init()
 
 AutoConfigFakeServer.fakeServer.autoRespond = true
 
-schemas = []
+schemas = [
+  FIXTURES['bower_components/appirio-tech-api-schemas/apiary/submissiondraft.json']
+];
 
-fixtures = []
```
Comment (vietnogi) [inline example/scripts/mock.coffee]: how come you took this check out?

Comment (samsep) [inline example/scripts/mock.coffee]: this always logs an error, though the data is there. This setup's also more consistent with all other repos and is simpler.

Comment (vietnogi) [inline example/scripts/mock.coffee]: the code looks like it should only log an error if a specified schema is undefined.  I added the code to give a warning if the specified mock dosnt exist.

File: example/scripts/mock.coffee [hunk 2]
```diff
@@ -2,7 +2,9 @@ AutoConfigFakeServer.init()
 
 AutoConfigFakeServer.fakeServer.autoRespond = true
 
-schemas = []
+schemas = [
+  'bower_components/appirio-tech-api-schemas/apiary/submissiondraft.json'
```
Comment (vietnogi) [inline example/scripts/mock.coffee]: do we need this?

Comment (samsep) [inline example/scripts/mock.coffee]: Yeah this is actually the ASP Application Service mock data but it's named submissiondraft. It includes the timeline mock data.

Comment (vietnogi) [inline example/scripts/mock.coffee]: oh, i see. just some minor code styles, otherwise lgtm

**Pull request merged**